src/helix-ui/elements/HXCheckboxElement.js

```diff
@@ -97,26 +87,24 @@ export class HXCheckboxElement extends HXElement {
         return this.hasAttribute('indeterminate');
     }
 
-    _onKeyUp (event) {
-        if (event.altKey) {
-            return;
-        }
-
-        if (event.keyCode === KEYS.Space) {
-            event.preventDefault();
-            this._toggleChecked();
+    set disabled (value) {
+        if (value) {
+            this.setAttribute('disabled', '');
+        } else {
+            this.removeAttribute('disabled');
         }
     }
 
-    _onClick () {
-        this._toggleChecked();
+    get disabled () {
+        return this.hasAttribute('disabled');
     }
 
-    _toggleChecked () {
-        if (this.disabled) {
-            return;
-        }
-        this.indeterminate = false;
-        this.checked = !this.checked;
+    _onChange (evt) {
+        this.checked = evt.target.checked;
+
+        // Chrome doesn't emit 'input' events for checkboxes and native 'change'
+        // events do not bubble out of the ShadowDOM. Emit a custom event to
+        // ensure a 'change' event makes it out of the ShadowDOM.
+        this.$emit('change');
```

Comment:
(1 of 2) Fix compatibility issue with regards to native `change` events not escaping ShadowDOM.